src/Raven.Server/Config/Categories/IndexingConfiguration.cs

```diff
@@ -222,6 +223,25 @@ public virtual bool RunInMemory
         [ConfigurationEntry("Indexing.NuGetPackageSourceUrl", ConfigurationEntryScope.ServerWideOnly)]
         public string NuGetPackageSourceUrl { get; set; }
 
+        [Description("Synchronization mode for the indexes")]
+        [DefaultValue(IndexWriteSynchronizationMode.Database)]
+        [IndexUpdateType(IndexUpdateType.None)]
+        [ConfigurationEntry("Indexing.JournalSynchronizationMode", ConfigurationEntryScope.ServerWideOnly)]
```

Comment:
Any reason we shouldn't make it a per db configuration?

Reply:
We could. But it makes a lot more sense to do that at the server level as that would give better behavior at the server level. I would say that unless we find the case where it really matters to control a single database write synchronization I would leave it Server wide.

Reply:
`ServerWideOnly` vs `ServerWideOrDatabase` still allows you to define it globally.
This matters, because db only configuration is something that you can do in the cloud without needing manual assitance.

Reply:
I also think it should be `ServerWideOrDatabase`